src/sql/parts/modelComponents/fileBrowserTree.component.ts

```diff
@@ -0,0 +1,142 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
+	Component, Input, Inject, ChangeDetectorRef, forwardRef, ViewChild, ElementRef, OnDestroy, AfterViewInit
+} from '@angular/core';
+
+import * as sqlops from 'sqlops';
+
+import { ComponentBase } from 'sql/parts/modelComponents/componentBase';
+import { IComponent, IComponentDescriptor, IModelStore, ComponentEventType } from 'sql/parts/modelComponents/interfaces';
+import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+import { FileBrowserViewModel } from '../fileBrowser/fileBrowserViewModel';
+import { FileNode } from 'sql/parts/fileBrowser/common/fileNode';
+import { FileBrowserTreeView } from '../fileBrowser/fileBrowserTreeView';
+
+@Component({
+	selector: 'modelview-fileBrowserTree',
+	template: `
+		<div #fileBrowserTree style="width:420px;height:700px"></div>
```

Comment:
why the hardcoded width and height?

Reply:
I tried using dimensions in %,  but it doesn't work since the container needs a specific height and width that the tree can take. Let me know if you have other ideas.